### ISSUE TITLE
Dispatcher keeps crashing and databse update errors

### ISSUE THREAD

**Issue by JanVogFZJ** [drolbr/Overpass-API#559]
The Overpass API becomes unavailable about once every day telling that either the overpassdispatcher or psql is not running. However, when I get this error, both _are_ running.
When restarting the dispatcher daemon the API works just as expected again though.
So this problem seems to be because the dispatcher daemon crashes somehow. However, the dispatcher does not generate any logs indicating that something went wrong, there is no error.

Are there any non-obvious log files the dispatcher daemon creates or is this a known bug?

Also, when running the command
`./apply_osc_to_db.sh $TEMP_DIR $FIRST_MINDIFF_ID --meta --augmented_diffs=no`
I get the following error hundreds of times per second with different node and way numbers:
> compute_geometry: Node 6359030131 used in way 34213806 not found.
What does this mean and how can I fix this?

P.S.: I installed the Overpass API using thi guide on an Ubuntu system: https://wiki.openstreetmap.org/wiki/Overpass_API/Installation#Applying_minutely_.28or_hourly.2C_or_daily.29_diffs

**Comment by mmd-osm** [comment]
> Node 6359030131 used in way 34213806 not found.

This means that your database is most likely corrupt, and you need to start from scratch again.
How did you create your database in the first place? Which Overpass version are you using?

**Comment by JanVogFZJ** [comment]
I just followed the guide I linked step by step.
I am using version 0.7.55.
A planet file.

**Comment by drolbr** [comment]
Are you sure you have running "psql"? There is no PostgreSQL involved in Overpass API.

Basically, it should be possible that both coexist on a machine, but it does not make much sense.

**Comment by JanVogFZJ** [comment]
Doesn't the typical "dispatcher not running" error say something with psql? I am not sure right now and can not test it as I am currently reimporting the data, but I am relatively sure it does mention psql somewhere.
But yes, Overpass does not use Postgres, that is correct. I have Postgres on the machine for the Nominatim Service that I run on the same machine.

**Comment by mmd-osm** [comment]
Did you delete your old database directory before starting a new import?

**Comment by JanVogFZJ** [comment]
Yes, this one and the replication-temp one were the only directories I deleted. Is that correct?

**Comment by mmd-osm** [comment]
Should be ok, yes.

**Comment by JanVogFZJ** [comment]
Same errors right after the initial import. So corrupt data is very unlikely.

**Comment by mmd-osm** [comment]
Can you provide an exact step-by-step list of the commands you've executed? It's extremely difficult for us to find out what exactly you're doing.

May I also suggest to follow this up on the dev mailing list: https://listes.openstreetmap.fr/wws/info/overpass, where you find other people running productive Overpass instances.

**Comment by JanVogFZJ** [comment]
This time I just used the ready-to-use container provided here: https://github.com/Frankkkkk/docker-overpass-api
The only command I executed was `make`.

**Comment by mmd-osm** [comment]
I'm afraid, we don't have any capacity to support third party Docker containers. You need to get in touch with the respective provider of those conatiners.

**Comment by JanVogFZJ** [comment]
I understand that, however I do not think its a problem of the container considering that the error is the exact same with and without the container.

**Comment by mmd-osm** [comment]
Look, that Docker container fetches the current master via` git clone https://github.com/drolbr/Overpass-API.git`, rather than using a released version on http://dev.overpass-api.de/releases/. This is not support, sorry.

**Comment by JanVogFZJ** [comment]
Ok, I get the same error with the latest **released** version (0.7.55) as well though.

And sorry, I did not get what I am supposed to do with the dev mailing list. Send an email where I describe the error I get?

**Comment by mmd-osm** [comment]
Yes, please include all relevant details and post that to the dev mailing list. You can browse the archive if you want to get an idea how this could look like.

**Comment by JanVogFZJ** [comment]
Ok, will do that!

**Comment by JanVogFZJ** [comment]
After reinstalling a clean version of Overpass using the official installaton guide (the one linked above seems to be an unofficial one) I did not have any more trouble (yet).
The Dispatcher is now working flawlessly for over a week.

**Comment by mmd-osm** [comment]
Ok thanks for your feedback. I'd suggest to close this issue, as there's nothing left to do.
